recipes-wpe/wpeframework/wpeframework-interfaces_git.bb

```diff
@@ -0,0 +1,48 @@
+SUMMARY = "WPEFramework interfaces"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=f1dffbfd5c2eb52e0302eb6296cc3711"
+PR = "r0"
+
+require include/wpeframework.inc
+DEPENDS = "wpeframework wpeframework-tools-native"
+
+SRC_URI = "git://github.com/rdkcentral/ThunderInterfaces.git;protocol=git;branch=master"
+SRCREV = "e17057ec4ae3d2f72dee9362d88e25c39fe4d2fa"
+
+# ----------------------------------------------------------------------------
+
+PACKAGECONFIG ?= "release"
+
+# Buildtype
+PACKAGECONFIG[debug]          = "-DBUILD_TYPE=Debug,,"
+PACKAGECONFIG[debugoptimized] = "-DBUILD_TYPE=DebugOptimized,,"
+PACKAGECONFIG[releasesymbols] = "-DBUILD_TYPE=ReleaseSymbols,,"
+PACKAGECONFIG[release]        = "-DBUILD_TYPE=Release,,"
+PACKAGECONFIG[production]     = "-DBUILD_TYPE=Production,,"
+
```

Comment:
Buildtype is only required in wpeframework code.

Reply:
Removed Buildtype.